### PR TITLE
18.09 merge

### PR DESCRIPTION
**Pull request by chiphuyen**
WIP: Transfer learning + sentiment analysis (IMDB + SST)

### CODE REVIEW

**Comment by okuchaiev** [comment]
I believe new PRs should be based on 18.10-dev now since 18.09 is "released". Please check with @vsl9 or @borisgin

File: requirements.txt [hunk 1]
```diff
@@ -8,5 +8,3 @@ mpi4py
 librosa==0.6.1
 matplotlib
 joblib==0.11
-sentencepiece
-sacrebleu
```
Comment (blisc) [inline requirements.txt]: Why are these removed?

File: test_error.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+def test():
```
Comment (blisc) [inline test_error.py]: What are these test scripts for?

File: requirements.txt (hunk 1 above)
Comment (okuchaiev) [reply requirements.txt]: we shouldn't remove these

**Comment by vsl9** [comment]
Sure, our current development branch is **18.10-dev**. So please make your PR against 18.10-dev.
